Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ApplyPaalmanPingsCorrection.py

```diff
@@ -65,7 +79,8 @@ def PyExec(self):
         prog_container.report('Starting algorithm')
 
         # Units should be wavelength
-        sample_unit = s_api.mtd[self._sample_ws_name].getAxis(0).getUnit().unitID()
+        sample_unit = \
+            s_api.mtd[self._sample_ws_name].getAxis(0).getUnit().unitID()
```

Comment:
I see, thanks! I somehow thought `flake8` would pick up the config file automatically but it doesn't seem to.